src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+spring.h2.console.enabled=true
+spring.datasource.username=postgres
+spring.datasource.password=Lv_2022_06
+spring.datasource.url=jdbc:postgresql://localhost:5432/SportHubUJP?createDatabaseIfNotExist=true
```

Comment:
you can add the next lines to generate tables automatically and you will find out if your entities are valid:

```
spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
spring.jpa.hibernate.ddl-auto=update
```